src/components/select/option/option.component.js

```diff
@@ -59,6 +59,12 @@ Option.propTypes = {
    * @ignore
    * True when option should be hidden from the view (prop added by the SelectList component) */
   hidden: PropTypes.bool,
+  /** MultiSelect only - custom Pill border color - provide any color from palette or any valid css color value. */
+  // eslint-disable-next-line react/no-unused-prop-types
+  borderColor: PropTypes.string,
+  /** MultiSelect only - fill Pill background with color */
+  // eslint-disable-next-line react/no-unused-prop-types
+  fill: PropTypes.bool,
```

Comment:
I am just wondering we are able to make some kind of validation props here? 🤔

Reply:
I think it would be doable but not in propTypes, it would have to be checked in the body of the component, i.e, Select and FilterableSelect would have to iterate through the Options and check if any of them have `fill` or `borderColor` prop and then thrown an error, not worth it in my opinion